homeassistant/components/cover/isy994.py

```diff
@@ -0,0 +1,115 @@
+"""
+Support for ISY994 covers.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.isy994/
+"""
+import logging
+from typing import Callable  # noqa
+
+from homeassistant.components.cover import CoverDevice, DOMAIN
+from homeassistant.components.isy994 import (ISYDevice, NODES, PROGRAMS, ISY,
```

Comment:
The way you import ISY here might cause issues

ISY is declared None in `components/isy994.py` so it is an immutable type. If you change it after the import (i.e. in during setup's `global ISY; ISY=xx` this ISY is still None

As I understand it you can do:
- `import homeassistant.components.isy999 as isy` and then refer to `isy.ISY` 
- or make ISY a mutable type like list or dict in `components/isy994.py`, then refer to ISY[0] or ISY['isy']